docs/user_guide/source/ecosystem/visualization/vtk.rst

```diff
@@ -0,0 +1,233 @@
+ 
+
+So far two "VTK File" types are supported:
+
+1. Image data (.vti)
+2. Unstructured Grid (.vtu)
+
+The main idea is to populate the above XML format contents describing the extent and the data arrays with ADIOS variables in the BP data set. The result is a more-than-compact typical VTK data file since extra information about the variable, such as dimensions and type, as they already exist in the BP data set.
+
+A typical VTK image data XML descriptor (.vti):
+
+.. code-block:: xml
+
+   <?xml version="1.0"?>
+   <VTKFile type="ImageData">
+     <ImageData WholeExtent="x1 x2 y1 y2 z1 z2" Origin="x0 y0 z0" Spacing="dx dy dz">
+       <Piece Extent="x1 x2 y1 y2 z1 z2">
+         <PointData>
+           <DataArray Name="p1"/>
+           <DataArray Name="p2"/>
+         </PointData>
+         <CellData>
+           <DataArray Name="c1"/>
+           <DataArray Name="c2"/>
+         </CellData>
+       </Piece>
+     </ImageData>
+   </VTKFile>
+   
+
+A typical VTK unstructured grid XML descriptor (.vtu):
+
+.. code-block:: xml
+
+   <?xml version="1.0"?>
+   <VTKFile type="ImageData">
+     <ImageData WholeExtent="x1 x2 y1 y2 z1 z2" Origin="x0 y0 z0" Spacing="dx dy dz">
+       <Piece Extent="x1 x2 y1 y2 z1 z2">
+         <PointData>
+           <DataArray Name="p1"/>
+           <DataArray Name="p2"/>
+         </PointData>
+         <CellData>
+           <DataArray Name="c1"/>
+           <DataArray Name="c2"/>
+         </CellData>
+       </Piece>
+     </ImageData>
+   </VTKFile>
+   
+
+In addition, VTK can interpret physical-time or output-step varying data stored with ADIOS by resusing the special "TIME" tag. This is better illustrated in the following section.
+
+
+Saving the vtk.xml data model
+-----------------------------
+
+For the full source code of the following illustration example see the `gray-scott adios2 tutorial <https://github.com/pnorbert/adiosvm/tree/master/Tutorial/gray-scott>`_
+
+To incorporate the data model in a BP data file, the application has two options: 
+
+1) Adding a string attribute called "vtk.xml" in code. "TIME" is a special tag for adding physical time variables.
+
+.. code-block:: c++
+
+   const std::string imageData = R"(
+        <?xml version="1.0"?>
+        <VTKFile type="ImageData" version="0.1" byte_order="LittleEndian">
+          <ImageData WholeExtent=")" + extent + R"(" Origin="0 0 0" Spacing="1 1 1">
+            <Piece Extent=")" + extent + R"(">
+              <CellData Scalars="U">
+                  <DataArray Name="U" />
+                  <DataArray Name="V" />
+                  <DataArray Name="TIME">
+                    step
+                  </DataArray>
+              </CellData>
+            </Piece>
+          </ImageData>
+        </VTKFile>)";
+
+   io.DefineAttribute<std::string>("vtk.xml", imageData);
+
+
+.. tip::
+
+   C++11 users should take advantage of the added string literal capabilities in C++11, R''( )",  as in the example above.
```

Comment:
C++11 users should take advantage C++11 string literals (R"( xml_here )") to simplify escaping characters in the XML.